authenticator/components/application-list-totp.js

```diff
@@ -0,0 +1,16 @@
+import React from 'react';
+import { StyleSheet, Text, View } from 'react-native';
+
+export default function ApplicationListTOTP( id ) {
+    return(
+        <View>
+            <Text style={styles.totpText}>000000</Text>
+        </View>
+    );
+};

```

Comment:
this is simple enough to add straight to the Application List Icon component instead of making an entirely new component. Especially since this seems like the only time we're using an Application List TOTP (in the front-end)